ask-sdk-core/ask_sdk_core/api_client.py

```diff
@@ -101,7 +100,11 @@ def _resolve_method(self, request):
             if invalid http request method is being called
         """
         try:
-            return getattr(requests, request.method.lower())
+            if request.method is not None:
+                return getattr(requests, request.method.lower())
+            else:
+                raise ApiClientException(
```

Comment:
redundant else?

Reply:
Sorry didn't get you. The `if .. else ..` was added to check since the `request` could have a `None` method as per it's structural definition. The `try .. catch.. ` is for the `getattr` call.